api/node/index.ts

```diff
@@ -361,6 +361,8 @@ export interface LoadFileOptions {
     libraryPaths?: Record<string, string>;
 }
 
+class SlintGlobal {}
```

Comment:
Why is this class needed?

Reply:
It is not needed. I will make the object an `Object`. Was an idea and I have forgotten to remove it.

Reply:
Removed